Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -1277,7 +1280,7 @@ - (void)testDownloadAlreadyOpenRealm {
         NSUInteger sizeBefore = fileSize(c.pathOnDisk);
         XCTAssertGreaterThan(sizeBefore, 0U);
         XCTAssertNotNil(RLMGetAnyCachedRealmForPath(c.pathOnDisk.UTF8String));
-        [self waitForExpectationsWithTimeout:10.0 handler:nil];
+        [self waitForExpectationsWithTimeout:15.0 handler:nil];
```

Comment:
15 seconds seems very long, even for large objects... did you arrive at this through experimentation? I'm not opposed to leaving it in if lower values sometimes cause CI failures, but I want to make sure this isn't leftovers from your debugging.